src/pages/api/projects/[id]/chapter.ts

```diff
@@ -40,12 +40,10 @@ export default APIWrapper({
       }
 
       const projectUpdate = req.body.projectUpdate as ChapterProjectUpdate;
-
-      const project = await updateChapterProject(
-        Types.ObjectId(projectId),
-        chapterId,
-        projectUpdate
-      );
+      const project = await updateChapterProject(Types.ObjectId(projectId), {
```

Comment:
Assuming we secured this endpoint to only allow chapter members and above to update a chapter project, what prevents a chapter member from updating another chapter's project?